config/7.10.0/envs/rhpam-authoring-ha.yaml

```diff
@@ -80,10 +80,12 @@ others:
                       value: "[[.ApplicationName]]-datagrid"
                     - name: SERVICE_PROFILE
                       value: "[[.ApplicationName]]-datagrid"
+                    - name: KUBERNETES_NAMESPACE
+                      value: "${KUBERNETES_NAMESPACE}"
+                    - name: KUBERNETES_LABELS
+                      value: "${KUBERNETES_LABELS}"
```

Comment:
do these env vars already exist within the running pod? If so, is it necessary to specify them here?

Reply:
no, they aren`t set on the images by default.